bookwyrm/views/notifications.py

```diff
@@ -15,16 +15,16 @@ def get(self, request, notification_type=None):
         """people are interacting with you, get hyped"""
         notifications = (
             request.user.notification_set.all()
-            .order_by("-created_date")
+            .order_by("-updated_date")
             .select_related(
                 "related_status",
-                "related_status__reply_parent",
+                "related_group",
+            )
+            .prefetch_related(
                 "related_import",
```

Comment:
This is wrong, it should be in `select_related` -- I'll write another unit test for that

Reply:
And I should retain `"related_status__reply_parent",`